### PR TITLE
Added a note to README about needing an AUTH_SECRET env var and warning to not commit this secret to repo

### PR DESCRIPTION
**Pull request by xunchuan-liu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,11 +33,13 @@ Actix provides the routing and error handling for actions that run through the H
 Bring up an endpoint:
 * `cargo run`: Starts an http server
 * Your `.env` file needs to define a couple things. The RUST_LOG config is optional depending on what you are looking to test/debug
+    * You need to generate a random string at least **48** bytes long for the AUTH_SECRET var in local development. Be sure this secret does **not** get commited anywhere in the repo.
```
Comment (xunchuan-liu) [inline README.md:36]: Editing .env variables to include AUTH_SECRET. Note to not commit this secret to any public repo.

Comment (trueb2) [reply README.md:36]: For things that produce the proper error message and check, I'd rather not add it to the README. Information not included in the error message that was produced was how to create a good/valid AUTH_SECRET.

File: README.md [hunk 2]
```diff
@@ -33,11 +33,13 @@ Actix provides the routing and error handling for actions that run through the H
 Bring up an endpoint:
 * `cargo run`: Starts an http server
 * Your `.env` file needs to define a couple things. The RUST_LOG config is optional depending on what you are looking to test/debug
+    * You need to generate a random string at least **48** bytes long for the AUTH_SECRET var in local development. Be sure this secret does **not** get commited anywhere in the repo. 
 ```
 RUST_LOG=asset_api=trace,rest_api=trace,actix=trace,actix_web=debug,diesel_migrations=trace
```
Comment (trueb2) [inline README.md:38]:
```suggestion
RUST_LOG=qsib_asset=trace,info
```

File: README.md (hunk 1 above)
Comment (trueb2) [inline README.md:36]:
```suggestion
    * `AUTH_SECRET` must be defined. Generate a random string at least **48** bytes long for local development.
    * Do **not** commit sensitive environment variables or tokens.
```

**Review by trueb2 — CHANGES_REQUESTED** [review]
Add suggestions for .env documentation.

**Review by trueb2 — APPROVED** [review]

**Pull request merged**